.github/workflows/ci.yml

```diff
@@ -125,21 +126,18 @@ jobs:
         run: |
           pip install -r requirements/requirements_docs.txt
 
+      - name: "Kill all servers"
+        uses: pyansys/pydpf-actions/kill-dpf-servers@v2.2
```

Comment:
@PProfizi I don't think these kill servers step are necessary, the workflow runs without it

Reply:
@cbellot000 I'll deal with that when refactoring the CI and actions.